app-catalog/src/components/charts/List.tsx

```diff
@@ -140,14 +145,47 @@ export function ChartsList({ fetchCharts = fetchChartsFromArtifact }) {
     );
   }
 
+  function SettingsLink() {
+    const theme = useTheme();
+
+    return (
+      <RouterLink routeName="/settings/plugins/app-catalog" tooltip="App-Catalog Settings">
+        <Typography
+          sx={{
+            size: '1rem',
+            marginLeft: '3rem',
+            color: theme.palette.text.primary,
+          }}
+        >
+          Settings
+        </Typography>
+      </RouterLink>
+    );
+  }
+
+  function AppCatalogTitle() {
```

Comment:
Can you please add stories for at least some of the main changes in this PR?

Reply:
done!